packages/botonic-plugin-contentful/tests/contentful/manage/manage-contentful-entry.test.ts

```diff
@@ -0,0 +1,74 @@
+import { CmsException, ContentType, TopContentId } from '../../../src/cms/'
+import { rndStr } from '../../../src/cms/test-helpers'
+import { ContentFieldType } from '../../../src/manage-cms/fields'
+import { ENGLISH } from '../../../src/nlp'
+import { repeatWithBackoff } from '../../../src/util/backoff'
+import { testContentful } from '../contentful.helper'
+import { ctxt, testManageContentful } from './manage-contentful.helper'
+
+describe('ManageContentful entries', () => {
+  test('TEST: createContent and deleteContent', async () => {
+    const contentful = testContentful({ disableCache: true })
+    const sut = testManageContentful()
+    const id = rndStr()
```

Comment:
use a locale variable for context to avoid hardcoding the language so many times
`const context = ctxt({locale: ENGLISH})`
I think you can also use this ManageContext  as a develivery Context where you do contentful.text()